### PR TITLE
Pass-Through command examples

### PR DESCRIPTION
**Pull request by dob71**

### CODE REVIEW

File: extras/lede_generic/pass-through-command-examples/blacklist.sh [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+echo "iptables -t filter -I FORWARD -j unum_forward" >> "$RULES_FILE"
+
+# Update UCI config to run the rules script on config changes
+if [ "$(uci set firewall.unum_blacklist)" != "include" ]; then
+uci set firewall.unum_blacklist=include
+uci set firewall.unum_blacklist.family=any
+uci set firewall.unum_blacklist.path=$RULES_FILE
+uci set firewall.unum_blacklist.reload=1
+uci set firewall.unum_blacklist.type=script
+uci commit
```
Comment (venkatchimata) [inline extras/lede_generic/pass-through-command-examples/blacklist.sh]: I think indentation is missing for this if block.

File: extras/lede_generic/pass-through-command-examples/blacklist.sh [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+fi
+
+# Apply the rules
+/etc/init.d/firewall restart
```
Comment (venkatchimata) [inline extras/lede_generic/pass-through-command-examples/blacklist.sh:43]: I am fully not sure about this comment. Just thinking if  restarting firewall would clear all the iptables rules added above. If we know that firewall restart does n't clear the existing iptables rules, I think this should be fine.

File: extras/lede_generic/pass-through-command-examples/dns_filter_update.sh [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+uci set firewall.unum_dns=include
+uci set firewall.unum_dns.family=any
+uci set firewall.unum_dns.path=$RULES_FILE
+uci set firewall.unum_dns.reload=1
+uci set firewall.unum_dns.type=script
+uci commit
+fi
+
+# Apply the rules
+/etc/init.d/firewall restart
```
Comment (venkatchimata) [inline extras/lede_generic/pass-through-command-examples/dns_filter_update.sh:108]: My comments for blacklist.sh apply for this one too.

File: extras/lede_generic/pass-through-command-examples/blacklist.sh (hunk 2 above)
Comment (dob71) [reply extras/lede_generic/pass-through-command-examples/blacklist.sh:43]: Note that the iptables commands are not executed, they are added to the /tmp/firewall_unum_blacklist. 
The file is referenced in the UCI settings for the firewall configuration.
When firewall is restarted the system applies the rules.
It's done that way to assure that the rules are not removed when the device is reconfigured from the web UI.

File: extras/lede_generic/pass-through-command-examples/dns_filter_update.sh (hunk 3 above)
Comment (dob71) [reply extras/lede_generic/pass-through-command-examples/dns_filter_update.sh:108]: indentation added

File: extras/lede_generic/pass-through-command-examples/blacklist.sh (hunk 1 above)
Comment (dob71) [reply extras/lede_generic/pass-through-command-examples/blacklist.sh]: indentation added

**Review by adamsmola — APPROVED** [review]

**Review by venkatchimata — APPROVED** [review]

**Pull request merged**